src/views/credits/credits.jsx

```diff
@@ -53,8 +48,8 @@ var Credits = React.createClass({
                     </li>
                     
                     <li>
-                        <img src="//cdn.scratch.mit.edu/get_image/user/2752403_170x170.png" alt="Saskia Avatar" />
-                        <span className="name">Saskia Leggett</span>
+                        <img src="//cdn.scratch.mit.edu/get_image/user/2752403_170x170.png" alt="Champika Avatar" />
```

Comment:
Champika's avatar image should be `//cdn.scratch.mit.edu/get_image/user/900283_170x170.png`, and since we list Scratch Team alphabetically by last name 'Fernando', should be between 'Carl Bowman' and 'Mark Ferrell'